src/app/ui/components/address-displayer/address-displayer.tsx

```diff
@@ -1,18 +1,16 @@
-import type { HTMLStyledProps } from 'leather-styles/types';
-
 import { isEven } from '@app/common/math/helpers';
 
 import { AddressDisplayerLayout } from './address-displayer.layout';
 import { groupByFour } from './address-displayer.utils';
 
-interface AddressDisplayerProps extends HTMLStyledProps<'span'> {
+interface AddressDisplayerProps {
   address: string;
 }
-export function AddressDisplayer({ address, ...props }: AddressDisplayerProps) {
+export function AddressDisplayer({ address }: AddressDisplayerProps) {
```

Comment:
It might seem redundant if they're not used, but these props let us style the element though it were a `styled` element, e.g now we can't add a margin directly to address displayer

Reply:
OK I can revert this. I prefer to add stuff like this when we need it rather than keep it in case we need it